lib/plugins/slack/attachements.js

```diff
@@ -108,9 +108,16 @@ module.exports = function(dataCollection, slackOptions) {
       }
     }
 
+    let message = '';
+    if (resultBaseURL) {
+      const baseDir = storageManager.getRelativeBaseDir();
```

Comment:
We have https://github.com/sitespeedio/sitespeed.io/blob/720d3b93c21034ce4ded3707916276a92888fb1a/lib/plugins/s3/index.js#L39 that makes the combo of S3 and outputFolder a combo that doesn't work. I think we should create another issue to fix this longterm